modules/node-shared/src/main/scala/org/tessellation/node/shared/infrastructure/snapshot/CurrencyMessageValidator.scala

```diff
@@ -89,7 +96,14 @@ object CurrencyMessageValidator {
           case _ if !isMetagraphIdValid =>
             Async[F].pure(WrongMetagraphId.invalidNec)
           case _ =>
-            validateSignatures(message)
+            for {
+              validSignature <- validateSignatures(message)
+              addressAlreadyUsed =
```

Comment:
It should be rather `addressNotUsed` because we have `validSignature` instead of `invalidSignature`

Reply:
👍🏼